Gemfile.lock

```diff
@@ -13,7 +13,10 @@ GEM
     nokogiri (1.11.1)
       mini_portile2 (~> 2.5.0)
       racc (~> 1.4)
+    nokogiri (1.11.1-java)
+      racc (~> 1.4)
     racc (1.5.2)
```

Comment:
gem/bundler updates this file automatically, 
it would have been added when I was testing things locally with `jruby`. 

I can take it out and see what happens... :)

Reply:
resolved here 29dd29f